app/Resources/assets/js/lsdoc/index.js

```diff
@@ -710,3 +710,21 @@ $(document).on('ready', adjustWindow);
 
 $(window).on('resize', adjustWindow);
 
+$(document).ready(function() {
+    var table = $('#datatable').DataTable();
+    $('#search_form_organization').on( 'keyup', function () {
+        table
+            .columns( 1 )
+            .search( this.value )
+            .draw();
+    });
+    $( '#search_form_user_role' ).on( 'change', function () {
+        if(this.value !== "") {
+            table.columns( 3 ).search( '(^'+this.value+'$)',true,false ).draw();
```

Comment:
`table.columns(3).search('(^'+this.value+'$)',true,false).draw();`
and on line 726